target_decisioning_engine/rule_evaluator.py

```diff
@@ -64,8 +65,10 @@ def process_rule(self, rule, context, request_type, request_detail, post_process
         tracer.trace_rule_evaluated(rule, context, rule_satisfied)
 
         if rule_satisfied:
-            if "key" in rule.get("consequence") or "state" in rule.get("consequence"):
+            if "key" in rule.get("consequence"):
                 create_view_response(deepcopy(rule.get("consequence")))
+            elif "state" in rule.get("consequence"):
+                create_prefetch_mbox_response(deepcopy(rule.get("consequence")))
```

Comment:
something is wrong in this logic - I don't see any "key" and "state" fields in rule consequences. For views, usually only the name (and not the key) is present, also we don't support "state" yet as we don't support any profile attrs/scripts in ODD atm.

Reply:
@XDex, Great - I was concerned we would lose data if I didn't use the right model, but sounds like I can just go with the simple approach of making all consequences conform to the MboxResponse type